src/main/kotlin/ink/abb/pogo/scraper/util/pokemon/PokemonData.kt

```diff
@@ -0,0 +1,22 @@
+/**
+ * Pokemon Go Bot  Copyright (C) 2016  PokemonGoBot-authors (see authors.md for more information)
+ * This program comes with ABSOLUTELY NO WARRANTY;
+ * This is free software, and you are welcome to redistribute it under certain conditions.
+ *
+ * For more information, refer to the LICENSE file in this repositories root directory
+ */
+
+package ink.abb.pogo.scraper.util.pokemon
+
+import POGOProtos.Data.PokemonDataOuterClass
+
+fun PokemonDataOuterClass.PokemonData.getIv(): Int {
```

Comment:
already defined in ink/abb/pogo/scraper/util/pokemon/Pokemon.kt:24 as well, maybe remove it there.